flink-libraries/flink-table/src/test/scala/org/apache/flink/table/descriptors/DescriptorPropertiesTest.scala

```diff
@@ -165,4 +182,26 @@ class DescriptorPropertiesTest {
       minLength,
       maxLength)
   }
+
+  private def testFixedFieldsValidation(properties: DescriptorProperties): Unit = {
```

Comment:
How about rename `testFixedFieldsValidation` to `testFixedIndexedPropertiesValidation`?